features/bootstrap/RaffleContext.php

```diff
@@ -248,6 +248,23 @@ public function weGetAnExceptionForARaffleWithNoComments()
         Assert::notNull($this->exceptionHappened);
     }
 
+    /**
+     * @Then winners comments should not be eligible for further raffling
+     */
+    public function winnersCommentsShouldNotBeEligibleForFurtherRaffling()
+    {
+        if (null !== $this->picked) {
+            $winnersComments = $this->picked->getComments();
+            $raffle          = $this->loadRaffle($this->raffleId);
+            foreach ($winnersComments as $comment) {
+//            if ($raffle->getCommentsEligibleForRaffling()->contains($comment)) {
+//                throw new Exception('It seems a comment is currently both eligible and non eligible.');
```

Comment:
This commented out code needed?

Reply:
no, of course not. that's why we have code reviews 👍